docs/modernization/WebViewCompatible.md

```diff
@@ -0,0 +1,224 @@
+---
+title: WebViewCompatible control for Windows Forms and WPF
+author: granitestatehacker
+description: The Windows Community Toolkit provides a version of the UWP web view control that can be used in WPF and Windows Forms applications. This control embeds a view into your application that renders web content in one of two ways. For client environments that support the WebViewControl (Windows 10), that implementation is used. For legacy systems, System.Windows.Controls.WebBrowser implements the view.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF
+---
+
+# WebViewCompatible controls for Windows Forms and WPF applications
+
+The **WebViewCompatible** controls show web content in your Windows Forms or WPF desktop application.
+
+![Web View Samples](../resources/images/Controls/WebView/web-view-samples.png)
+
+These controls use the one of two rendering engines.  For Windows 10 client environments, the newer Microsoft Edge rendering engine is used to embed a view that renders richly formatted HTML content from a remote web server, dynamically generated code, or content files.  For legacy systems, the System.Windows.Controls.WebBrowser is used, providing Internet Explorer engine-based rendering.
```

Comment:
>legacy systems [](start = 276, length = 14)

More specific language here--should be a requirements section or something that distills down what behavior happens in what circumstance